### PR TITLE
feat: sort country names

### PR DESCRIPTION
**Pull request by lucianaDilizio**
Sort country names

### CODE REVIEW

File: src/components/form-helpers/form-helpers.js [hunk 1]
```diff
@@ -39,6 +39,15 @@ function translateIntlTelInputCountryNames(language) {
...
   }
+  countryData.sort((a, b) => {
+    if (a.name.toLocaleLowerCase() < b.name.toLocaleLowerCase()) {
+      return -1;
+    }
+    if (a.name.toLocaleLowerCase() > b.name.toLocaleLowerCase()) {
+      return 1;
+    }
+    return 0;
+  });
```
Comment (andresmoschini) [inline src/components/form-helpers/form-helpers.js]: Is it possible adding a test for it?

File: src/components/form-helpers/form-helpers.js [hunk 2]
```diff
@@ -39,6 +39,15 @@ function translateIntlTelInputCountryNames(language) {
           : nameInCurrentLanguage;
     }
   }
+  countryData.sort((a, b) => {
+    if (a.name.toLocaleLowerCase() < b.name.toLocaleLowerCase()) {
```
Comment (andresmoschini) [inline src/components/form-helpers/form-helpers.js]: What do you think about using [this approach](https://stackoverflow.com/questions/8996963/how-to-perform-case-insensitive-sorting-in-javascript#49004987)?

```
[ 'Foo', 'bar' ].sort((a, b) => a.localeCompare(b, undefined, { sensitivity: 'base' }));
```

Maybe some browsers do not support it, but in that case, it is not so bad.

File: src/components/form-helpers/form-helpers.js (hunk 1 above)
Comment (lucianaDilizio) [reply src/components/form-helpers/form-helpers.js]: Applied 😄

File: src/components/Signup/Signup.test.js [hunk 3]
```diff
@@ -35,5 +35,69 @@ describe('Signup', () => {
 
     // Assert
     expect(container.querySelectorAll('.error')).not.toHaveLength(0);
+  });  
+
+  it('should show Argentina below Argelia', () => {
```
Comment (andresmoschini) [inline src/components/Signup/Signup.test.js]: What do you think about 

```suggestion
  it('should show Argentina below Argelia when selected language is es', () => {
```

File: src/components/Signup/Signup.test.js [hunk 4]
```diff
@@ -35,5 +35,69 @@ describe('Signup', () => {
...
+          </Router>
+        </DopplerIntlProvider>
+      </AppServicesProvider>,
+    );
+     
+    // Assert
+    expect(container.querySelector('#iti-item-dz').nextElementSibling.id).toBe('iti-item-ar');
+  });  
+
+  it('should show Territorio Palstino, Ocupado below Territorio Britanico del Oceano Indico', () => {
```
Comment (andresmoschini) [inline src/components/Signup/Signup.test.js]:
```suggestion
  it('should show Territorio Palestino, Ocupado below Territorio Britanico del Oceano Indico', () => {
```

Comment (andresmoschini) [reply src/components/Signup/Signup.test.js]: What do you think about also indicate the language in the test name?

File: src/components/Signup/Signup.test.js [hunk 5]
```diff
@@ -35,5 +35,69 @@ describe('Signup', () => {
...
+          </Router>
+        </DopplerIntlProvider>
+      </AppServicesProvider>,
+    );
+     
+    // Assert
+    expect(container.querySelector('#iti-item-io').nextElementSibling.id).toBe('iti-item-ps');
+  });
+
+  it('should show American Samoa below Algeria', () => {
```
Comment (andresmoschini) [inline src/components/Signup/Signup.test.js]:
```suggestion
  it('should show American Samoa below Algeria when language is en', () => {
```

File: src/components/Signup/Signup.test.js [hunk 6]
```diff
@@ -35,5 +35,69 @@ describe('Signup', () => {
...
+          </Router>
+        </DopplerIntlProvider>
+      </AppServicesProvider>,
+    );
+     
+    // Assert
+    expect(container.querySelector('#iti-item-dz').nextElementSibling.id).toBe('iti-item-as');
+  });
+
+  it('should show Brunel below British Indian Ocean', () => {
```
Comment (andresmoschini) [inline src/components/Signup/Signup.test.js]:
```suggestion
  it('should show Brunel below British Indian Ocean when language is en', () => {
```

**Comment by andresmoschini** [comment]
Great work @lucianaDilizio ! I really like those tests!

File: src/components/Signup/Signup.test.js (hunk 4 above)
Comment (lucianaDilizio) [reply src/components/Signup/Signup.test.js]: I'm agree. Thanks!

**Pull request merged**